### DISCUSSION TITLE
你该如何为 Kubernetes 定制特性 - 面向信仰编程 · /cloud-native-kubernetes-extension

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#641] [Announcements]
https://draveness.me/cloud-native-kubernetes-extension/

**Comment by zcong1993** [comment]
感觉只能仰望一下相关开源项目了，作为普通人一般也不需要定制化扩展🤣

**Comment by draveness** [comment]
> @zcong1993 感觉只能仰望一下相关开源项目了，作为普通人一般也不需要定制化扩展🤣

在公司内部做 Kubernetes 的话，可能会经常接触到 CRD、准入控制和调度器

**Comment by zcong1993** [comment]
> @draveness
> > @zcong1993 感觉只能仰望一下相关开源项目了，作为普通人一般也不需要定制化扩展🤣
> 
> 在公司内部做 Kubernetes 的话，可能会经常接触到 CRD、准入控制和调度器

好的，着重关注下这几点，之前只是用了些开源的 operator

**Comment by kissjacky** [comment]
板凳

**Comment by pytimer** [comment]
目前用到了CRD、准入控制，其他两个再学习下

**Comment by piamo** [comment]
mark,学习了

**Comment by woofyzhao** [comment]
controller/operator官方是否有一个template proj方便扩展用户直接快速搭建boilerplates

**Comment by draveness** [comment]
> @woofyzhao
> controller/operator官方是否有一个template proj方便扩展用户直接快速搭建boilerplates

sample-controller
